src/viewport/index.tsx

```diff
@@ -667,6 +664,20 @@ class Viewport extends React.Component<
         );
     }
 
+    public startRecording(): void {
+        if (!this.recorder) {
+            return;
+        }
+        this.recorder.start();
+    }
+
+    public stopRecording(): void {
+        if (!this.recorder) {
+            return;
+        }
+        this.recorder.stop();
+    }
+
```

Comment:
Handle recording with these instead of passing down a boolean from the front end